Svenbjorn/whatson
# Feature/activate location

Monday: https://visitfaroeislands.monday.com/boards/4732496989/pulses/5106392545
Activate location on mapbox has a button in whatson map view

## How to test

Works for all whatson elements.

 - View in phone/tablet view
 - Open map view with the bottom-right menu
 - See that location button is shown on the bottom-left
 - Click it to allow location permission
 - The map should then get your current location to show.
